src/types/libdef/npm-custom/@testing-library/react_v11.x.x.js

```diff
@@ -399,54 +615,59 @@ declare module '@testing-library/react' {
   // dom-testing-library re-declares
   declare export function queryByTestId(
     container: HTMLElement,
-    id: TextMatch,
-    options?: TextMatchOptions
+    id: Matcher,
+    options?: MatcherOptions
   ): ?HTMLElement;
   declare export function getByTestId(
     container: HTMLElement,
-    id: TextMatch,
-    options?: TextMatchOptions
+    id: Matcher,
+    options?: MatcherOptions
   ): HTMLElement;
   declare export function queryByText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: TextMatchOptions
+    text: Matcher,
+    options?: MatcherOptions
   ): ?HTMLElement;
   declare export function getByText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: { selector?: string, ... } & TextMatchOptions
+    text: Matcher,
+    options?: { selector?: string, ... } & MatcherOptions
   ): HTMLElement;
   declare export function queryByPlaceholderText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: TextMatchOptions
+    text: Matcher,
+    options?: MatcherOptions
   ): ?HTMLElement;
   declare export function getByPlaceholderText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: TextMatchOptions
+    text: Matcher,
+    options?: MatcherOptions
   ): HTMLElement;
   declare export function queryByLabelText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: TextMatchOptions
+    text: Matcher,
+    options?: MatcherOptions
   ): ?HTMLElement;
   declare export function getByLabelText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: { selector?: string, ... } & TextMatchOptions
+    text: Matcher,
+    options?: { selector?: string, ... } & MatcherOptions
   ): HTMLElement;
   declare export function queryByAltText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: TextMatchOptions
+    text: Matcher,
+    options?: MatcherOptions
   ): ?HTMLElement;
   declare export function getByAltText(
     container: HTMLElement,
-    text: TextMatch,
-    options?: TextMatchOptions
+    text: Matcher,
+    options?: MatcherOptions
   ): HTMLElement;
   declare export function getNodeText(node: HTMLElement): string;
   declare export var screen: Screen<>;
+
+  // ADDED BY JULIEN
+  declare export function getDefaultNormalizer(
+    options?: $Shape<{| trim: boolean, collapseWhitespace: boolean |}>
+  ): NormalizerFn;
```

Comment:
This was missing from the libdef: this was the primary reason why I updated the libdef... and I was disappointed to see it wasn't in the new one either!
I should contribute it probably...